tests/RobotFramework/tests/cumulocity/log/log_operation_plugins.robot

```diff
@@ -220,6 +220,85 @@ Agent resilient to plugin dirs removal
     ThinEdgeIO.Service Health Status Should Be Up    tedge-agent
     Should Have MQTT Messages    c8y/s/us    date_from=${date_from}    message_pattern=118,
 
+Plugin include filter
+    # Clean up any existing plugins for test isolation
+    Execute Command    rm /usr/share/tedge/log-plugins/file
+    Execute Command    rm /usr/share/tedge/log-plugins/journald
+
```

Comment:
Why `dmsg` is not removed in this test?
```
Execute Command    rm /usr/share/tedge/log-plugins/dmesg
```

and if it's forgotten, probably it's better to create a key, like `Cleanup existing plugins` since these removals repeat in several tests.

Reply:
Resolved in the squashed commit bd7c2a95c0